source/common/quic/platform/quiche_flags_constants.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "source/common/http/utility.h"
+
+#define OVERRIDDEN_RELOADABLE_FLAGS(KEY_VALUE_PAIR)                                                \
+  KEY_VALUE_PAIR(quic_disable_version_draft_29,                                                    \
+                 true) /* Envoy only supports RFC-v1 in the long term, so disable IETF draft 29    \
+                          implementation by default. */                                            \
+  KEY_VALUE_PAIR(                                                                                  \
+      quic_default_to_bbr,                                                                         \
+      true) /* This flag enables BBR, otherwise QUIC will use Cubic which is less performant */
+
+#define OVERRIDDEN_PROTOCOL_FLAGS(KEY_VALUE_PAIR)                                                  \
```

Comment:
Do we intend to add these (or similar macros) to the platform API? How would we keep these in sync across platform implementations?

Or is this just in quic/platform for unrelated reasons?

Reply:
I don't think the overridden flag list should be part of platform API. They are Envoy specific. But it is supposed to be imported into our internal code base as the source of truth. So that we can define these macros internally to manipulate the flag values according to our own flag implementation.